pipes/WDL/tasks/tasks_reports.wdl

```diff
@@ -392,8 +392,13 @@ task align_and_count {
     File   ref_db
     Int    topNHits = 3
 
+    Boolean? filter_bam_to_proper_primary_mapped_reads         = false
```

Comment:
linters don't like this -- if it's set to a default value, no need to call it an optional input, it can just be required